export async function handleError(error: Error) {
    console.error("Error", error);
}
